2021-04-01 16:56:16,043 INFO: Match startup [in A:\projects\match\app\__init__.py:47]
2021-04-01 16:57:36,616 INFO: Match startup [in A:\projects\match\app\__init__.py:45]
2021-04-01 17:07:00,598 INFO: Match startup [in A:\projects\match\app\__init__.py:45]
2021-04-01 17:09:14,046 INFO: Match startup [in A:\projects\match\app\__init__.py:45]
2021-04-01 17:13:23,587 INFO: Match startup [in A:\projects\match\app\__init__.py:45]
2021-04-01 17:16:52,534 INFO: Match startup [in A:\projects\match\app\__init__.py:45]
2021-04-01 17:17:56,148 INFO: Match startup [in A:\projects\match\app\__init__.py:45]
2021-04-01 17:19:31,910 INFO: Match startup [in A:\projects\match\app\__init__.py:45]
2021-04-01 17:21:06,738 INFO: Match startup [in A:\projects\match\app\__init__.py:45]
2021-04-01 17:24:03,541 INFO: Match startup [in A:\projects\match\app\__init__.py:45]
2021-04-01 23:00:09,701 INFO: Match startup [in A:\projects\match\app\__init__.py:45]
2021-04-01 23:01:06,441 INFO: Match startup [in A:\projects\match\app\__init__.py:45]
2021-04-01 23:04:52,810 INFO: Match startup [in A:\projects\match\app\__init__.py:45]
2021-04-01 23:17:55,142 INFO: Match startup [in A:\projects\match\app\__init__.py:47]
2021-04-01 23:24:29,195 INFO: Match startup [in A:\projects\match\app\__init__.py:49]
2021-04-02 15:58:13,288 INFO: Match startup [in A:\projects\match\app\__init__.py:82]
2021-04-02 15:58:32,203 ERROR: Exception on /auth/reset_password_request [POST] [in a:\projects\match\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "a:\projects\match\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "a:\projects\match\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "a:\projects\match\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "a:\projects\match\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "a:\projects\match\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "a:\projects\match\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "A:\projects\match\app\auth\routes.py", line 61, in reset_password_request
    send_password_reset_email(user)
  File "A:\projects\match\app\auth\email.py", line 7, in send_password_reset_email
    token = user.get_reset_password_token()
  File "A:\projects\match\app\models.py", line 110, in get_reset_password_token
    return jwt.encode(
AttributeError: 'str' object has no attribute 'decode'
2021-04-02 16:19:45,735 INFO: Match startup [in A:\projects\match\app\__init__.py:82]
2021-04-02 17:25:36,079 INFO: Match startup [in A:\projects\match\app\__init__.py:82]
2021-04-02 17:49:37,902 INFO: Match startup [in A:\projects\match\app\__init__.py:82]
2021-04-02 17:49:43,603 INFO: Match startup [in A:\projects\match\app\__init__.py:82]
2021-04-02 17:50:36,911 INFO: Match startup [in A:\projects\match\app\__init__.py:82]
2021-04-02 17:50:42,125 INFO: Match startup [in A:\projects\match\app\__init__.py:82]
